scripts/validate-markdown-values

```diff
@@ -3,13 +3,12 @@
 set -e -o pipefail -u
 
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+root_dir=$(cd $PWD &>/dev/null; pwd -P)
```

Comment:
Hmmm I'd rather actually use
```suggestion
root_dir=$(cd "$self_dir/.." &>/dev/null; pwd -P)
```
Coz `$PWD` (current working directory) does NOT need to be the root directory of the project when someone executes this script